pkg/bindings/system/system.go

```diff
@@ -36,7 +36,7 @@ func Events(ctx context.Context, eventChan chan entities.Event, cancelChan chan
 	if cancelChan != nil {
 		go func() {
 			<-cancelChan
-			err = response.Body.Close()
+			err := response.Body.Close()
 			logrus.Errorf("Unable to close event response body: %v", err)
```

Comment:
Is logrus smart enough to not report an error if err==nil?

Reply:
Unfortunately, it isn't; that was something I noticed already in several places throughout the podman code base. I would be perfectly happy to "if" the logrus call as it causes annoying log messages that might make things difficult to operators.

Reply:
yes this needs an if err != nil block around it

Reply:
done